Person data batch export FAQ (#8489)
PostHog/posthog.com: contents/docs/cdp/batch-exports/index.md

- Some features of the old export destinations are still being ported over.
  - This includes logs and error reporting.
 

> ## How can I export person info?
> 
> Currently it's not possible, but it's on the roadmap:
> - [person data export](https://github.com/PostHog/posthog/issues/20339).
> - Export person properties on events (`event.person_properties`) with [custom schemas](https://github.com/PostHog/posthog/issues/20367).

Reviewer: ```suggestion
## How can I export person profiles?

It's not currently possible to export person profiles and properties, but it is on the roadmap:

- [Person data export](https://github.com/PostHog/posthog/issues/20339).
- Export person properties on events (`event.person_properties`) with [custom schemas](https://github.com/PostHog/posthog/issues/20367).
```

> Note: Our implementation of person ids on events (`event.person_id`) is eventually consistent, which for batch export destinations means joining event `distinct_id` with the person data export.

Reviewer: Why is this important? `event.person_id` is included in batch exports now?
```suggestion
> **Note:** Our implementation of person IDs on events (`event.person_id`) is eventually consistent. For batch export destinations, this means it requires joining event `distinct_id` with the person data export.
```

Author: Because people will internally will update person_ids (overrides table + squash) and they will never get that update and that will be confusing. 
On the other hand persons data export will send them updates, e.g. a new distinct_id was added here.

> Note: Our implementation of person ids on events (`event.person_id`) is eventually consistent, which for batch export destinations means joining event `distinct_id` with the person data export.

Reviewer 2: I would omit this. It's not wrong it's just that we don't have a person data export (as stated above). Eventually, once we develop the person model this will be the way to go. As it stands, this note seems confusing.

Author: I need it to answer why we're not exporting `person_id` as that's the user question

- Some features of the old export destinations are still being ported over.
  - This includes logs and error reporting.
 
## How can I export person profiles?

It's not currently possible to export person profiles and properties, but it is on the roadmap:

- [Person data export](https://github.com/PostHog/posthog/issues/20339).
- Export person properties on events (`event.person_properties`) with [custom schemas](https://github.com/PostHog/posthog/issues/20367).

> > **Note:** Our implementation of person IDs on events (`event.person_id`) is eventually consistent. For batch export destinations, this means it requires joining event `distinct_id` with the person data export.

Author: ```suggestion
> **Note:** Our implementation of person IDs on events (`event.person_id`) is eventually consistent, so its not exportable.
```
thanks Tomas

Reviewer 2: Thank you
